### ISSUE TITLE
camera.getLivePreview is not supported

### ISSUE THREAD

**Issue by gshcherb** [Insta360Develop/Insta360_OSC#7]
SDK v1.3.10 is not supporting `camera.getLivePreview` command. Can this be added?

**Comment by qizhiyu2018** [comment]
use SDK instead of OSC to get LivePreview

**Comment by gshcherb** [comment]
Yes, we can potentially use SDK for this. But there are number of problems with using SDK:
 - it is basically doubles size of APK with for us and brings additional dependencies, that we ideally can skip adding. Since both One X and One X2 we are targeting support on-device stitching, it should be possible to downsize the preview images on camera stitch it at near runtime and send back to the client side when requested by `camera.getLivePreview`.
 - there are multiple cameras that we can potentially support and preferred ones are those that adhere to OSC specification, because we can have one logic and interaction implementation for different brands and different types of cameras. If we start adding SDK for every specific camera size and complexity can quickly get out of control.

**Comment by qizhiyu2018** [comment]
Dynamic loading SDK, only when user use Insta360 cameras they download these files.

**Comment by gshcherb** [comment]
Even if we implement dynamic delivery for the Insta SDK this anyways means separate implementation for every camera brand, when everything can be implement on OSC side. Still highly desirable to have as much OSC functionality as possible.

**Comment by MaksimDanilov** [comment]
Hello! As I see, there is currently no real-time preview implementation for OSC. Will this be done in the near future?

**Comment by antonhudz** [comment]
Hello Dear @le-insta360 @nibtod , any plans to bring livePreview to OSC?


Ricoh Theta cameras already support it via OSC for example https://github.com/ricohapi/theta-api-specs/blob/main/theta-web-api-v2.1/commands/camera.get_live_preview.md

Thank you 🙏

**Comment by wagner1343** [comment]
Why insta360 doesn't implements OSC for camera preview?
